### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by amanpro30**
Fixes some grammatical errors

### CODE REVIEW

**Comment by amanpro30** [comment]
@googlebot I signed it!

**Comment by kevmoo** [comment]
Running here https://dart-review.googlesource.com/c/sdk/+/136803

File: CHANGELOG.md [hunk 1]
```diff
@@ -2247,7 +2247,7 @@ Still need entries for all changes to dart:html since 1.x
 
 #### `dart:isolate`
 
-*   Make `Isolate.spawn` take a type parameter representing the argument type of
+*   Make `Isolate.spawn` takes a type parameter representing the argument type of
```
Comment (kevmoo) [inline CHANGELOG.md]: revert this change – it was fine as-is
